src/account/accountReducers.js

```diff
@@ -79,9 +79,8 @@ export default (state = initialState, action) => {
       ];
     }
     case LOGOUT: {
-      const newState = state.slice();
-      newState[0].apiKey = '';
-      return newState;
+      // Remove the active account
```

Comment:
Hmm, yeah, that feature I can see wanting, because "logout and login again" is a common thing to do when trying to deal with apparent bugs.